pom.xml

```
@@ -3,4 +3,54 @@
   <groupId>com.chbrown13.rec_test</groupId>
   <artifactId>tool-recommender-bot-test</artifactId>
```

Hi, the latest version of Python changes ```print``` to a built-in function instead of a statement, leading to a PEP 3105 warning here [1]. We recommend changing this line to ```suggestion
print({s})```. This change will not impact the functionality of your code. Additionally, Python is officially no longer supporting Python 2 as of Jan. 1, 2020 [2]. Please consider upgrading the code for your project to Python 3. Thanks!

[1] https://www.python.org/dev/peps/pep-3105/
[2] https://www.python.org/doc/sunset-python-2/